lib/renderer/webview/webview.h

```diff
@@ -1881,20 +1904,36 @@ struct webview_priv
       {
         goto error_dlg;
       }
-      if (dlg->lpVtbl->GetResult(dlg, &res) != S_OK)
-      {
-        goto error_dlg;
-      }
-      if (res->lpVtbl->GetDisplayName(res, SIGDN_FILESYSPATH, &ws) != S_OK)
-      {
-        goto error_result;
+      if (flags & WEBVIEW_DIALOG_FLAG_MULTIPLE) {
+        if (dlg->lpVtbl->GetResults(dlg, &multires) != S_OK) {
+          goto error_dlg;
+        }
```

Comment:
@leaanthony I tried for like hours and I couldn't get it work in windows. I think its panic-ing at `GetResults`. See if you can help me with windows implementation. I tried my best

Reply:
I think I found the issue and boy, it's subtle! Look what the docs say about GetResult:
![Screen Shot 2021-01-17 at 6 21 42 am](https://user-images.githubusercontent.com/1943904/104820967-a2819880-588c-11eb-8089-dd3a9d062af2.png)
Looks like we need to use `GetResults()`!

Reply:
I think you missed little `s` :). Indeed I am using `GetResults`. 

I think `GetResults` is method of interface `IFileOpenDialog` not `IFileDialog`. But I actually tried with `IFileOpenDialog` also in my local (by replacing all instances of `IFileDialog` with `IFileOpenDialog`), still same issue happened (I am not sure the way i did is right).


If you think its too complicated and we're better off without this feature, I am totally OK with it. We can drop this PR. I found https://github.com/ncruces/zenity which can be used to select multiple files. I am using it in my project without an issue.